engineapi/backup_monitor.go

```diff
@@ -59,14 +66,15 @@ func NewBackupMonitor(logger logrus.FieldLogger,
 
 	// Call engine API snapshot backup
 	if backup.Status.State == longhorn.BackupStateNew {
-		_, replicaAddress, err := engineClient.SnapshotBackup(backup.Name, backup.Spec.SnapshotName,
+		engineBackupName, replicaAddress, err := engineClient.SnapshotBackup(backup.Name, backup.Spec.SnapshotName,
```

Comment:
It's good to see we already have backward compatibility for this, so we can have this simple solution here.